src/mobu/services/business/tap.py

```diff
@@ -173,15 +170,12 @@ def _make_client(self, token: str) -> pyvo.dal.TAPService:
         if not config.environment_url:
             raise RuntimeError("environment_url not set")
         tap_url = str(config.environment_url).rstrip("/") + "/api/tap"
-        try:
-            s = requests.Session()
-            s.headers["Authorization"] = "Bearer " + token
-            auth = pyvo.auth.AuthSession()
-            auth.credentials.set("lsst-token", s)
-            auth.add_security_method_for_url(tap_url, "lsst-token")
-            auth.add_security_method_for_url(tap_url + "/sync", "lsst-token")
-            auth.add_security_method_for_url(tap_url + "/async", "lsst-token")
-            auth.add_security_method_for_url(tap_url + "/tables", "lsst-token")
-            return pyvo.dal.TAPService(tap_url, auth)
-        except Exception as e:
-            raise TAPClientError(e, user=self.user.username) from e
+        s = requests.Session()
+        s.headers["Authorization"] = "Bearer " + token
+        auth = pyvo.auth.AuthSession()
+        auth.credentials.set("lsst-token", s)
+        auth.add_security_method_for_url(tap_url, "lsst-token")
+        auth.add_security_method_for_url(tap_url + "/sync", "lsst-token")
+        auth.add_security_method_for_url(tap_url + "/async", "lsst-token")
+        auth.add_security_method_for_url(tap_url + "/tables", "lsst-token")
+        return pyvo.dal.TAPService(tap_url, auth)
```

Comment:
That try/catch block is there from hard-won experience, I fear. This change will cause it to throw random things like `ValueError` and `KeyError` with absolutely no useful information about what precisely failed. It's there to get the context of "this happened while creating a TAP client" into the exception along with the underlying exception type and message.

Reply:
There's a `phase: make_client` tag set here (and a span created), but it's not easy to see because it's in the `startup` method instead.  I moved it into this method, does that make this better? You could see this in the issue in the Sentry UI. We could also put the `phase` tag in the sentry notification (though it's still not as clear as putting it in the exception message itself).